src/nodedef.cpp

```diff
@@ -1437,6 +1593,43 @@ void CNodeDefManager::resetNodeResolveState()
 	m_pending_resolve_callbacks.clear();
 }
 
+void CNodeDefManager::mapNodeboxConnections()
+{
+	for (u32 i = 0; i < m_content_features.size(); i++) {
+		ContentFeatures *f = &m_content_features[i];
+		if ((f->drawtype != NDT_NODEBOX) && (f->node_box.type != NODEBOX_CONNECTED))
+			continue;
+		for (std::vector<std::string>::iterator it = f->connects_to.begin();
+				it != f->connects_to.end(); ++it) {
+			std::set<content_t> ids;
+			getIds(*it, ids);
+			for (std::set<content_t>::iterator k = ids.begin();
+					k != ids.end(); ++k)
+				f->connects_to_ids.insert(*k);
```

Comment:
It look like these 5 lines can be simplified to just: `getIds(*it, f->connects_to_ids);`